tests/docker/globus.el8.Dockerfile

```diff
@@ -0,0 +1,88 @@
+FROM almalinux:8
+
+RUN \
+  dnf update -y && \
+  dnf install -y \
+    pam-devel \
+    python3-jsonschema \
+    epel-release \
+    gcc-c++ \
+    gnupg \
+    make \
+    python3 \
+    python3-pip \
+    rsyslog \
+    sudo \
+    wget \
+    which \
+    diffutils \
+    procps \
+    rpm-build \
+    authd \
+    unixODBC-devel \
+  && \
+  dnf clean all && \
+  rm -rf /var/cache/dnf /tmp/*
+
+# python 2 and 3 must be installed separately because dnf will ignore/discard python2
+RUN \
+  dnf check-update -q >/dev/null || { [ "$?" -eq 100 ] && dnf update -y; } && \
+  dnf install -y \
+    python3 \
+    python3-devel \
+    python3-pip \
+  && \
+  dnf clean all && \
+  rm -rf /var/cache/dnf /tmp/*
+
+RUN python3 -m pip --no-cache-dir install xmlrunner distro psutil pyodbc jsonschema requests
+
+RUN dnf install -y \
+        dnf-plugin-config-manager \
+    && \
+    rpm --import https://packages.irods.org/irods-signing-key.asc && \
+    dnf config-manager -y --add-repo https://packages.irods.org/renci-irods.yum.repo && \
+    dnf config-manager -y --set-enabled renci-irods && \
+    rpm --import https://core-dev.irods.org/irods-core-dev-signing-key.asc && \
+    dnf config-manager -y --add-repo https://core-dev.irods.org/renci-irods-core-dev.yum.repo && \
+    dnf config-manager -y --set-enabled renci-irods-core-dev && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf /var/cache/yum /tmp/*
+
+#### Get and install globus repo ####
+RUN wget -q https://downloads.globus.org/globus-connect-server/stable/installers/repo/rpm/globus-repo-latest.noarch.rpm && \
+    rpm --force -i globus-repo-latest.noarch.rpm
+
+#### Install and configure globus specific things ####
+RUN dnf install -y globus-gridftp-server-progs \
+    globus-simple-ca \
+    globus-gass-copy-progs \
+    globus-gsi-cert-utils-progs \
+    globus-proxy-utils
+
+RUN dnf --disablerepo epel install -y globus-common-devel \
+    globus-gridftp-server-devel \
+    globus-gridmap-callout-error-devel
+
+RUN mkdir /iRODS_DSI
+RUN chmod 777 /iRODS_DSI
```

Comment:
Collapse these into a single `RUN` block. If this pattern is repeated elsewhere, do the same to them.

Reply:
done